gradle.properties

```diff
@@ -0,0 +1,3 @@
+version=4.7.0-SNAPSHOT
+hivemq-api.version=4.6.2
```

Comment:
shouldnt this all the the same version for the plugin to work? am i missing sth?